client/src/views/home/LoginPage.vue

```diff
@@ -26,6 +26,8 @@
             v-model="password"
             @on-enter-keyup="onLoginClick()"
             id="ms-password-input"
+            :is-wrong="props.error ? true : false"
+            :error-message="$t('HomePage.organizationLogin.passwordError')"
```

Comment:
Set `error-message` only if `props.error` is set, you don't need `is-wrong`.

Reply:
Done